src/contents/ui/VolumeBar.qml

```diff
@@ -19,20 +18,32 @@ Item {
 
         CommandIcon {
             size: iconSize;
-            onClicked: container.changeVolume(container.volume - 0.1 > 1 ? 1 : container.volume - 0.1)
+            onClicked: player.changeVolume(-volumeStep / 100, false)
             source: 'audio-volume-low';
         }
         Rectangle {
             MouseArea {
                 anchors.centerIn: parent
-                height: parent.height + 8
+                height: iconSize
                 width: parent.width
                 cursorShape: Qt.PointingHandCursor
+                property int wheelDelta: 0
                 onClicked: ({x}) => {
-                    container.changeVolume(x/parent.width)
+                    player.setVolume(x/parent.width)
```

Comment:
I'm not a qml expert, I learned it while working on this project and had never used it before, so maybe I'm wrong, but I don't really like the fact that in qml you can refer to components that are defined in other files, that's why I used signals and properties to make the various components more isolated.

IMHO this approach has many advantages:
- Isolated components are easier to manage and update.
- Reduced interdependencies help to avoid unintended side-effects when changes are made.
- Components are more flexible and more easily reusable.

Reply:
I think you're right. Will address this and your other suggestions later today.

Thanks.

Reply:
There are three signals now

```
signal setVolume(newVolume: real)
signal volumeUp()
signal volumeDown()
```